### PR TITLE
docs: Update Readme about watchOS support

### PR DESCRIPTION
**Pull request by philipphofmann**
## :scroll: Description

See Readme.

## :bulb: Motivation and Context

We don't want our users to expect that Sentry is working on watchOS as it is on the other platforms.

## :pencil: Checklist

- [x] I reviewed submitted code
- [x] No breaking changes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
         <img src="https://sentry-brand.storage.googleapis.com/sentry-logo-black.png" width="280">
     </a>
 <br/>
-    <h1>Official Sentry SDK for iOS / tvOS / macOS / watchOS <sup>(1)</sup>.</h1>
+    <h1>Official Sentry SDK for iOS / tvOS / macOS / partially watchOS <sup>(1)</sup>.</h1>
```
Comment (marandaneto) [inline README.md:6]: L: what's about using `limited` instead? so it matches the explanation below

File: README.md [hunk 2]
```diff
@@ -63,7 +63,7 @@ SentrySDK.start(options: [
 
 For more information checkout the [docs](https://docs.sentry.io/platforms/cocoa/?platform=swift).
 
-<sup>(1)</sup>limited symbolication support
+<sup>(1)</sup>limited symbolication support and no crash handling.
```
Comment (marandaneto) [inline README.md:66]: M: what's about raising a PR for https://docs.sentry.io/platforms/cocoa/?platform=swift#install too? I'm all in favor of a single source of truth, but this is very important.
this comment doesn't block this PR btw.

File: README.md (hunk 1 above)
Comment (philipphofmann) [reply README.md:6]: I also like limited more.

File: README.md (hunk 2 above)
Comment (philipphofmann) [reply README.md:66]: Good point, I'm going to change this as well.

Comment (philipphofmann) [reply README.md:66]: https://github.com/getsentry/sentry-docs/pull/1924

**Pull request merged**